parsl/executors/taskvine/executor.py

```diff
@@ -970,43 +905,47 @@ def _taskvine_submit_wait(task_queue=multiprocessing.Queue(),
                 if t is None:
                     task_found = False
                     continue
-                # When a task is found:
-                executor_task_id = vine_id_to_executor_task_id[str(t.id)]
-                logger.debug("Completed TaskVine task {}, executor task {}".format(t.id, executor_task_id))
-                result_file = result_file_of_task_id.pop(executor_task_id)
+                logger.debug('Found a task!')
```

Comment:
don't use `!` surprise in log messages - if it's a surprise it should be a WARNING